segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/UtilsWrapper.java

```diff
@@ -253,4 +258,48 @@
                             .thenApplyAsync(vv -> infoList, chunkedSegmentStorage.getExecutor()),
                 chunkedSegmentStorage.getExecutor()), streamSegmentName);
     }
+
+    /**
+     * Performs sanity operations on chunk like create chunk, write to the chunk, check if the chunk exists, read back contents to the chunk and delete the chunk.
+     *
+     * @param chunkName Name of the chunk.
+     * @param dataSize dataSize of the bytes to read.
+     * @return A CompletableFuture that, when completed, will indicate that the operation completed.
+     * If the operation failed, it will be completed with the appropriate exception. Notable Exceptions:
+     * {@link ChunkStorageException} In case of I/O related exceptions.
+     */
+    public CompletableFuture<Void> checkChunkSegmentStorageSanity(String chunkName, int dataSize) {
+        Preconditions.checkNotNull(chunkName, "chunkName");
+        Preconditions.checkArgument(chunkName.length() > 0, "chunkName");
```

Comment:
I suggest we use
`Preconditions.checkArgument(!Strings.isNullOrEmpty(chunkName), "chunk name must not be null or empty");`